src/components/header/_macro-options.md

```diff
@@ -1,21 +1,22 @@
-| button          | object`<SignOutButton>`                           | false                            | Settings for the [sign out button](#signoutbutton) in the header used to exit a transactional service                                 |
-| navigation      | array`<Navigation>`                               | false                            | Settings for the [main menu links](#navigation)                                                                                       |
+| Name              | Type                                              | Required                         | Description                                                                                                                           |
+| ----------------- | ------------------------------------------------- | -------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| phase             | `PhaseBanner` [_(ref)_](/components/phase-banner) | false                            | Settings to set the Phase banner component within the HTML `<header>` element                                                         |
+| wide              | boolean                                           | false                            | Set to “true” to increase the maximum width of the layout container to 1280px                                                         |
+| fullWidth         | boolean                                           | false                            | Set to “true” to increase the maximum width of the layout container to the full width of the viewport                                 |
+| classes           | string                                            | false                            | Classes to add to the wrapping `header`                                                                                               |
+| variants          | array or string                                   | false                            | An array of values or single value (string) to adjust the component using available variants: “internal”, "neutral" and “description” |
+| mastheadLogoUrl   | string                                            | false                            | Wraps the masthead logo in a link. Set the URL for the HTML `href` attribute for the link.                                            |
+| mastheadLogo      | object`<MastheadLogo>`                            | false                            | Settings for a [custom organisation logo](#mastheadlogo) in the masthead. Defaults to the ONS logo.                                   |
+| mastheadLogoMulti | object`<MastheadLogoMulti>`                       | false                            | Settings for multiple logos in masthead                                                                                               |
```

Comment:
Had another thought about the `mastheadLogoMulti` param. We have introduced this to avoid breaking changes, deprecating `mastheadLogo`. But I dont think that is the correct thing to do because we would then have to do another breaking change when we renamed this back to `mastheadLogo`. I think the best way to do this would have `mastheadLogo` also accept the params that `mastheadLogoMulti` accepts then at a later date remove the `large` and `small` params